aptoide-ui/build.gradle.kts

```diff
@@ -28,15 +28,14 @@ android {
   buildFeatures {
     // Enables Jetpack Compose for this module
     compose = true
+    buildConfig = true
```

Comment:
No need in this, default is `true`

Reply:
Apparently on the the new AGP 8.0 the default value has been changed to false. I had to change it to true.

https://developer.android.com/reference/tools/gradle-api/8.0/com/android/build/api/dsl/BuildFeatures#buildConfig()